### PR TITLE
(CONT-1144) - Conversion of ERB to EPP templates

### PR DESCRIPTION
**Pull request by praj1001**
## Summary
Conversion of ERB to EPP templates for haproxy module.

## Checklist
- [ ] 🟢 Spec tests.
- [ ] 🟢 Acceptance tests.
- [ ] Manually verified. (For example `puppet apply`)

### CODE REVIEW

File: lib/puppet/functions/haproxy/generate_error_message.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
```
Comment (Ramesh7) [inline lib/puppet/functions/haproxy/generate_error_message.rb:1]: Please clean-up the default comments and update with required details.

File: lib/puppet/functions/haproxy/sort_bind.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+# https://github.com/puppetlabs/puppet-specifications/blob/master/language/func-api.md#the-4x-api
```
Comment (Ramesh7) [inline lib/puppet/functions/haproxy/sort_bind.rb:3]: same as above can we please remove unwanted comments and add based on usecase?

File: lib/puppet/functions/haproxy/validate_ip_addr.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'ipaddr'
+# https://github.com/puppetlabs/puppet-specifications/blob/master/language/func-api.md#the-4x-api
```
Comment (Ramesh7) [inline lib/puppet/functions/haproxy/validate_ip_addr.rb:4]: same as above.

File: spec/functions/generate_error_message_spec.rb [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'haproxy::generate_error_message' do
+  # please note that these tests are examples only
```
Comment (Ramesh7) [inline spec/functions/generate_error_message_spec.rb:6]: same here.

**Review by jordanbreen28 — APPROVED** [review]

**Pull request merged**